src/components/application_manager/include/application_manager/app_launch/app_launch_data_db.h

```diff
@@ -36,13 +36,10 @@
 #include "application_manager/app_launch/app_launch_data_impl.h"
 #include "application_manager/app_launch_settings.h"
 #include "utils/macro.h"
-#ifdef __QNX__
-#include "utils/qdb_wrapper/sql_database.h"
-#include "utils/qdb_wrapper/sql_query.h"
-#else  // linux
-#include "utils/sqlite_wrapper/sql_database.h"
-#include "utils/sqlite_wrapper/sql_query.h"
-#endif
+
+#include "sql_database.h"
```

Comment:
@BSolonenko Use this rule: [Order of includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/commit/a3a73a446d1a8fe30f7b98dde719b35719b6d731